neo/public/stylesheets/main.css

```diff
@@ -131,22 +131,35 @@ search-bar {
 }
 
 .small-text{
-  font-size: 15px;
-}
-
-.big-text{
-  font-size: 30px;
+  font-size: 13px;
 }
 
-#countDiv{
+#count-div{
   color: #000000;
-  margin-bottom: 30px;
+  margin-bottom: 40px;
   font-weight:500;
   float: left;
+  margin-top: 20px;
+  padding-right:20px;
+}
+
+#count-div > p{
```

Comment:
yes, I wanted remove all of their margin-bottom to make two lines of text closer.